### DISCUSSION TITLE
immich_db_dumper => Prune (keep last ...)

### DISCUSSION THREAD

**Discussion by MrColumbo** [immich-app/immich#7251] [Q&A]
Today I moved my immich installation and did an export/dump of the database and it worked quite well.
I learned about immich_db_dumper and i was wondering if there are parameters i can pass to only keep the last 3 days or whatever I like to keep. I saw that I can pass the frequency like daily but if I do daily backups i do not like to keep them forever.

**Comment by mmomjian** [comment]
You can set `BACKUP_KEEP_DAYS=3` as an environment variable for the immich_db_dumper. See [here](https://github.com/prodrigestivill/docker-postgres-backup-local)
